src/DIRAC/__init__.py

```diff
@@ -121,9 +121,11 @@
 alarmMail = "dirac.alarms@gmail.com"
 
 # Set rootPath of DIRAC installation
-
-pythonPath = os.path.realpath(__path__[0])
-rootPath = os.path.dirname(pythonPath)
+if six.PY3:
+  rootPath = sys.base_prefix  # pylint: disable=no-member
+else:
+  pythonPath = os.path.realpath(__path__[0])
+  rootPath = os.path.dirname(pythonPath)
```

Comment:
IIUC the most visible change here will be the location of the /etc/dirac.cfg file.

At a very minimum, we'll need to write some instructions.
And there is also a tutorial on performing integration tests on the development machine, and that should be updated.

Reply:
Yeah, the current place the `dirac.cfg` file and the `grid-security` directory ends up is definitely wrong (`lib/pythonX.Y/site-packages/etc/`).

I'm not sure what the "right" solution is, this is in incoherent dump but...
1. `diracos/etc/`: Cleaner, but gets deleted every time there is an update
2. `diracos/../etc/`: What happens when you install a client with `pip` or `conda` directly?
3. `diracos/etc/` and symlink to one level higher: Better but we have to make multiple symlinks, one for `dirac.cfg`, one for `etc/grid-security/certificates`, one for `etc/grid-security/voms`, one for `etc/grid-security/vomsdir`, ...
4. `diracos/etc/dirac/` which is a symlink to a stable location
5. `diracos/etc/` but let it always be overridden by an environment variable (`$DIRAC_ETC_DIR`)
6. Lots of combinations of the above

I'm tempted towards (5) and this would also make it easier to have a shared installation on CVMFS. I'm also undecided about if we want to set the X509 variables as part of this.

Reply:
I don't love any of the above but (5) seems like the best. (3) and (4) are possible but messy and error-prone. For reference on the env variables: https://dirac.readthedocs.io/en/latest/AdministratorGuide/ServerInstallations/environment_variable_configuration.html

Which takes us to the fact that right now we have a bashrc file acting as a collector of env variables, and those should not be lost. Is diracos/diracosrc sourcing other *rc files?

Reply:
Following up in https://github.com/DIRACGrid/DIRAC/issues/5082